charts/incubator/firefox/values.yaml

```diff
@@ -1,29 +1,48 @@
 image:
   repository: tccr.io/truecharts/firefox
   pullPolicy: IfNotPresent
-  tag: 101.0.1@sha256:ed6178aa78fa1d11cad7c86d6554b3d9e903a82aec65ab6d7fb8ea77aeb6810b
+  tag: v109.0.1@sha256:0d5c436afbd776aab406ec12ff2230fa45b4aaba31f75735cca0cde4254fc7ec
 
 securityContext:
-  runAsNonRoot: false
-  readOnlyRootFilesystem: false
-
-podSecurityContext:
-  runAsUser: 0
-  runAsGroup: 0
+  container:
+    readOnlyRootFilesystem: false
+    runAsNonRoot: false
+    runAsUser: 0
+    runAsGroup: 0
 
 service:
   main:
     ports:
       main:
-        targetPort: 3000
+        protocol: http
         port: 10131
 
+workload:
+  main:
+    podSpec:
+      containers:
+        main:
+          probes:
+            liveness:
+              type: http
+            readiness:
+              type: http
+            startup:
+              type: http
```

Comment:
probes need to be adjusted when basic auth is added

Reply:
honestly...im ok removing basic auth...it defaults to abc for username and if no password is set, disabled.